### PR TITLE
[MER-2252] add assets_required input

### PR DESCRIPTION
**Pull request by doratzeng**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ jobs:
       with:
         assets: test_results/output.xml
         aviator_api_token: ${{ secrets.AVIATOR_API_TOKEN }}
+        assets_required: true  # NOTE: This is optional (the default value is false). If true, the action will fail if no assets are found.
```
Comment (jainankit) [inline README.md]: let's do default to true? That way this is proactively detecting it but users have a way to turn it off?

File: upload.sh [hunk 2]
```diff
@@ -22,8 +22,13 @@ for filename in ${ASSETS}; do
 done
 
 if [ "${#all_files[@]}" -eq 0 ]; then
-    echo "No files found."
-    exit 1
+    if [ "${ASSETS_REQUIRED}" = "true" ]; then
+        echo "No files found."
```
Comment (jainankit) [inline upload.sh]:
```suggestion
        echo "No asset files found to upload."
```

File: upload.sh [hunk 3]
```diff
@@ -22,8 +22,13 @@ for filename in ${ASSETS}; do
...
+        exit 1
+    else
+        echo "WARNING: No files found."
```
Comment (jainankit) [inline upload.sh]:
```suggestion
        echo "WARNING: No asset files found to upload."
```

File: README.md [hunk 4]
```diff
@@ -46,7 +46,7 @@ jobs:
       with:
         assets: test_results/output.xml
         aviator_api_token: ${{ secrets.AVIATOR_API_TOKEN }}
-        assets_required: true  # NOTE: This is optional (the default value is false). If true, the action will fail if no assets are found.
+        assets_required: false  # NOTE: This is optional (the default value is true). If true, the action will fail if no assets are found.
```
Comment (jainankit) [inline README.md]: Let's do this as a multi-line comment

**Review by jainankit — APPROVED** [review]

**Comment by Iftytasu1** [comment]
[Uploading admin.zip…]()

**Pull request merged**